lib/makeHttpRequest.js

```diff
@@ -36,6 +36,7 @@ var makeHttpRequest = function(method, url, contentType, body) {
     .send(body)
     .set('Content-Type', 'application/json; charset=utf-8')
     .set('X-XSRF-Protected', '1')
+    .timeout(5000)
```

Comment:
@jeffheifetz isn't this timeout for connections to the camera? I'd assume the camera would be on the local network, so the user's location shouldn't matter